public/ripgrep/build

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -e
+
+cargo build --release
+make DESTDIR="$1" install
```

Comment:
Why running `make install` when the repository does not contain any `Makefile`? I think you should just copy the binary in the `/bin` or `/usr/bin` folder of `$1`

Reply:
I use Rust occasionally, so I don't know why I'm `make`ing. `install` is a good route. Will fix.